pkgs/applications/version-management/mercurial/default.nix

```diff
@@ -180,7 +180,7 @@ in
         buildInputs = self.buildInputs ++ self.propagatedBuildInputs;
         nativeBuildInputs = self.nativeBuildInputs;
 
-        phases = [ "installPhase" "installCheckPhase" ];
```

Comment:
I think we need todo something with the none standard `installCheckPhase`.

Reply:
is it really? https://nixos.org/manual/nixpkgs/stable/#ssec-controlling-phases
> ### 6.5.1.1.1. phases
Specifies the phases. You can change the order in which phases are executed, or add new phases, by setting this variable. If it’s not set, the default value is used, which is
$prePhases unpackPhase patchPhase $preConfigurePhases configurePhase $preBuildPhases buildPhase checkPhase $preInstallPhases installPhase fixupPhase **installCheckPhase** $preDistPhases distPhase $postPhases